lib/graphql/plug/endpoint.ex

```diff
@@ -9,16 +9,18 @@ defmodule GraphQL.Plug.Endpoint do
       {mod, func} -> apply(mod, func, [])
       s -> s
     end
-    %{schema: schema}
+    root_value = Keyword.get(opts, :root_value, %{})
+    %{root_value: root_value, schema: schema}
```

Comment:
Needed to be alpha in order for pattern matching to work.
```
iex(1)> %{zoo: "a", alpha: 1}
%{alpha: 1, zoo: "a"}
```
It actually gets re-ordered automatically. This took me about 2 hours working on before I gave up, went to bed, then woke up at 4am and had that aha moment.